Scripts/Modules/TooltipModule/TooltipUI.cs

```diff
@@ -1,6 +1,8 @@
 ﻿#if UNITY_EDITOR
-using TMPro;
 using System;
+#if INCLUDE_TEXT_MESH_PRO
+using TMPro;
+#endif
```

Comment:
Please also include the CCU attribute

[assembly: OptionalDependency("TMPro.TextMeshProUGUI", "INCLUDE_TEXT_MESH_PRO")]

Also, are we sure the change below won't break serialization? Can you test locally by removing TMP from your project and then bringing it back? Maybe we were getting away with serialization not breaking before because TMP was causing compile errors.

Reply:
Added and pushed